README.md

```diff
@@ -21,9 +21,9 @@
   - Aggregating key metrics, such as associated revenue, related to each user's interactions with individual campaigns and flows (and organic actions)
   - Aggregating these metrics further, to the grain of campaigns, flows, and individual users
 
-The following table provides a detailed list of all models materialized within this package by default. 
+The following table provides a detailed list of all models materialized within this package by default.  
 
-| **Model**                | **Description**                                                                                                                                |
+| **Model**                |  **Description**                                                                                                                                 |
```

Comment:
Just wanted to call out here that the models in the table below seem to be left overs from the copied package. I know there is a final clean up task, but may be worth updating this section as you build the models.

Reply:
Cleaned! thanks 👍

Reply:
Quick request to lowercase the T in the model names.

Reply:
It looks like this was rather removed entirely. Was that intended?

Reply:
![image](https://github.com/fivetran/dbt_twilio/assets/74217849/8178c796-f740-4228-a007-f76f13329b80)

Reply:
my bad-- thanks for catching

Reply:
It looks like the names of the models in the table don't include the double underscores. Other than that, this looks good to go.

Reply:
Thanks for adding the double underscores. However, can you also underscore the T in the names to be consistent with  the model name casing.